src/raftstore/store/peer.rs

```diff
@@ -599,6 +640,13 @@ impl Peer {
         self.get_store().committed_index() == self.get_store().applied_index()
     }
 
+    #[inline]
+    pub fn ready_to_handle_read(&self) -> bool {
+        // If committed_index doesn't equal to applied_index, written apply state may be overwritten
```

Comment:
The comments doesn't match the code, it's possible that the `committed_index != applied_index` while `applied_index_term == self.term()`.